memgpt/models/pydantic_models.py

```diff
@@ -13,6 +13,14 @@
 from memgpt.utils import get_human_text, get_persona_text, get_utc_time
 
 
+class OptionState(str, Enum):
```

Comment:
@sarahwooders I added a new type here for kwargs that are "optional booleans", basically yes, no, or "I don't care"/default

Reply:
I figured there's probably some standardized way for doing this or standardized type in `typing` (not just `Optional[bool]`) or Pydantic that I could use, but I couldn't find anything